Tests/ContentTypeTest.swift

```diff
@@ -16,13 +16,13 @@ final class ContentTypeTests: XCTestCase {
         XCTAssertEqual(endPoint.pathComponent, "content_types")
     }
     #if !NO_FATAL_TEST
-    func testFetch_withoutUID() async {
-        expectFatalError(expectedMessage: "Please provide ContentType uid") {
-            makeContentTypeSut().fetch { (result: Result<AssetModel, Error>, response) in
-                        
-            }
-        }
-    }
+//    func testFetch_withoutUID() async {
```

Comment:
@vkalta Could you please clarify the reason for commenting out the test cases block?

Reply:
This is outdated code @sunil-lakshman . This needs to run in the main thread else an error is thrown. So will revisit it later.